libs/barista-components/secondary-nav/src/secondary-nav.scss

```diff
@@ -207,5 +236,5 @@ $dt-secondarynav-header-padding-left: 20px;
 .dt-secondary-nav-section
   .dt-secondary-nav-section-header.dt-secondary-nav-section-expanded:hover
   .dt-secondary-nav-section-title {
-  color: $gray-700;
+  color: $gray-700; // ask ryan
```

Comment:
Let's ask Ryan or @dynatrace-oss/ux-core and figure out if this is the correct value.

Reply:
Removed comment, verified color matches value in invision mockup